plasmapy/diagnostics/thomson.py

```diff
@@ -653,7 +653,7 @@ def _spectral_density_model(wavelengths, settings=None, **params):
     electron_vel = electron_speed[:, np.newaxis] * electron_vdir
     ion_vel = ion_speed[:, np.newaxis] * ion_vdir
 
-    # Convert temperatures from eV to Kelvin (required by fast_spectral_density)
+    # Convert temperatures from eV to kelvin (required by fast_spectral_density)
```

Comment:
Wasn't it correct capitalised?

Reply:
I thought so too, but apparently not: https://en.wikipedia.org/wiki/Kelvin#:~:text=The%20word%20%22kelvin%22%20is%20not,K%20is%20a%20capital%20letter.